src/main/java/gregicadditions/machines/TileEntityProcessingArray.java

```diff
@@ -221,8 +227,11 @@ protected Recipe findRecipe(long maxVoltage,
 												  .EUt(recipe.getEUt())
 												  .duration(recipe.getDuration());
 
-			copyChancedItemOutputs(newRecipe, recipe, minMultiplier);
-			newRecipe.notConsumable(this.machineItemStack);
+			//Don't allow MV or LV macerators to have chanced outputs, because they do not have the slots for chanced outputs
+			if(!(mte instanceof MetaTileEntityMacerator && (((MetaTileEntityMacerator) mte).getTier() == 1 || ((MetaTileEntityMacerator) mte).getTier() == 2))) {
```

Comment:
I think he hard-coded it because macerators are unique in this respect. No other machine I'm aware of has a different number of slots based on machine tier.

In theory it would be more robust to look up how many output slots there are, but I'm not sure whether such a mechanism exists.

Reply:
Checking if the number of output slots is <= 1 would not work because of some recipes in GTCE, such as the Carbon fibers, which are done in the autoclave, which has 1 output slot. Voiding if the number of output slots was <= 1 would mean that this recipe would always fail.

Since the macerator is the only machine that has a variable number of output slots, checking directly if the machine was a macerator was the best way that I could think of to take this into account.

Reply:
fair enough. there might be some other way of checking by inspecting the recipe map, but it really isn't worth the effort